### ISSUE TITLE
Insufficient color contrast

### ISSUE THREAD

**Issue by nichgray** [CDRH/austen#73]
Text colors do not have sufficient contrast. There are many examples from the home page results, but here is one: 

<img width="518" height="31" alt="Image" src="https://github.com/user-attachments/assets/1a38ce5f-58ec-414f-8d62-19d504b1d670" />

**Comment by erinchambers** [comment]
`#dd2c2f` should work as a good replacement!

**Comment by erinchambers** [comment]
Fixed with PR #88
